oracle/src/events/processAMBCollectedSignatures/index.js

```diff
@@ -5,14 +5,20 @@ const bridgeValidatorsABI = require('../../../../contracts/build/contracts/Bridg
 const rootLogger = require('../../services/logger')
 const { web3Home, web3Foreign } = require('../../services/web3')
 const { signatureToVRS, packSignatures } = require('../../utils/message')
+const { readAccessListFile } = require('../../utils/utils')
 const { parseAMBMessage } = require('../../../../commons')
 const estimateGas = require('./estimateGas')
 const { AlreadyProcessedError, IncompatibleContractError, InvalidValidatorError } = require('../../utils/errors')
 const { MAX_CONCURRENT_EVENTS, EXTRA_GAS_ABSOLUTE } = require('../../utils/constants')
 
 const limit = promiseLimit(MAX_CONCURRENT_EVENTS)
 
-const { ORACLE_ALWAYS_RELAY_SIGNATURES } = process.env
+const {
+  ORACLE_HOME_TO_FOREIGN_ALLOWANCE_LIST,
+  ORACLE_HOME_TO_FOREIGN_BLOCK_LIST,
+  ORACLE_HOME_TO_FOREIGN_CHECK_SENDER,
```

Comment:
this parameter was introduced for the vanilla bridge to provide an ability to turn on/off a costly check of the sender. In the AMB case, this check is chip since the sender is stored in the AMB message. Consider to remove this logic from AMB code.

Reply:
Done in 2e53a71